src/test/java/com/coutemeier/maven/plugins/asciidoctor/lifecycle/ThemeMojoIT.java

```diff
@@ -38,4 +38,21 @@ public void noThemesConfigured()
             && this.validator.dependencyNotExists()
         );
     }
+
+    @Test
+    public void themeBuildDirectoryDoesntExistsTest()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/asciidoctor-lifecycle-maven-plugin/asciidoctor-lifecycle-maven-plugin/pullRequest?prid=5396310)